src/Compiler/Microsoft.AspNetCore.Razor.Language/test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/ComplexTagHelpers_Runtime.codegen.cs

```diff
@@ -493,15 +493,8 @@ public async System.Threading.Tasks.Task ExecuteAsync()
 #line hidden
 #nullable disable
 #nullable restore
-#line (26,25)-(26,26) "TestFiles/IntegrationTests/CodeGenerationIntegrationTest/ComplexTagHelpers.cshtml"
-@
-
-#line default
-#line hidden
-#nullable disable
-#nullable restore
-#line (26,26)-(26,49) "TestFiles/IntegrationTests/CodeGenerationIntegrationTest/ComplexTagHelpers.cshtml"
-DateTimeOffset.Now.Year
```

Comment:
If we had been validating C# diagnostics, this would have been caught before FUSE was merged. More priority to https://github.com/dotnet/razor/issues/10247.